content/blog/2017/08/2017-08-23-pull-requests-and-more.adoc

```diff
@@ -0,0 +1,123 @@
+the link:https://plugins.jenkins.io/git[git plugin],
+the link:https://plugins.jenkins.io/git-client[git client plugin],
+and is a technical evangelist for link:http://cloudbees.com[CloudBees, Inc.]
+====
+
+The Jenkins project booth at link:https://www.cloudbees.com/jenkinsworld/home[Jenkins World 2017]
+will include the "Pull Requests Corner", recruiting new Jenkins contributors.
+We think there are many people who will attend the conference without realizing how easy it is
+to help the Jenkins project, and how much the help is appreciated.
+
+Meet us in the "Pull Requests Corner" and we'll help you find a way to help Jenkins.
+Here are some areas where we can use your help.
+Most of them do not require coding, and do not require a large time commitment.
+
+=== One Minute Feedback on Your Version
+
+The Jenkins changelog pages (link:https://jenkins.io/changelog-stable/[LTS] and link:https://jenkins.io/changelog-stable/[weekly]) gather user experiences with specific Jenkins versions.
+You can help other Jenkins users by clicking one of the weather icons in the link:https://jenkins.io/changelog-stable/[LTS changelog] (or the link:https://jenkins.io/changelog-stable/[weekly changelog]) for the release you're using.
+Changelog feedback from weekly releases helps the release team select the long term support version.
+Changelog feedback from LTS releases helps other users prepare to upgrade.
+
+It takes less than a minute, and helps the community (which will ultimately help you).
+
+=== Five Minutes to Answer a Question
+
+image:/images/logos/needs-you/Jenkins_Needs_You-02.png["Jenkins needs YOU", role="right", width=480] 
+In five minutes or less, you can help other Jenkins users.
+
+For example:
+
+* Answer a question on the #jenkins channel of link:http://webchat.freenode.net/?channels=jenkins[internet relay chat (IRC)]
+* Answer a question on the link:https://groups.google.com/forum/#!forum/jenkinsci-users[Jenkins users mailing list]
+* Answer a question on link:https://www.reddit.com/r/jenkinsci/[reddit]
+
+=== Ten Minutes to Learn and Share
+
+If you have ten minutes, you can learn something new and share what you learned.
+
+* Watch a link:https://www.youtube.com/watch?v=FhDomw6BaHU&list=PLvBBnHmZuNQJsTCaXs91HRrmso7RNSl-L[Jenkins Minute video] and tweet what you learned
+* Read a link:https://jenkins.io/node/[Jenkins community blog post] and share what you learned on LinkedIn
+* Read a link:https://devops.com/?s=Jenkins[devops.com blog post] and share it on facebook
+* Read a link:https://www.cloudbees.com/blog[CloudBees blog post] and test drive an idea from it
+* Find another Jenkins blog and share it (I recently discovered link:http://cicd.life/tutorials/[Matt Bajor's ci/cd life blog])
+
+=== Fifteen Minutes for Pipeline
+
+Liam Newman has created the "link:https://www.youtube.com/watch?v=FhDomw6BaHU&list=PLvBBnHmZuNQJsTCaXs91HRrmso7RNSl-L[Jenkins Minute]" video series.
```

Comment:
This is the same tip as the first link under "10 minutes".

I suggest dropping the 'watch a Jenkins Minute video' bullet from the 10 minutes, and leaving it here